src/components/POIForm.component.js

```diff
@@ -1,8 +1,189 @@
+        ["1"]
+      )
+    });
+  }*/
+
+  onChangeLinks(inputLinks) {
+    this.setState({
+      links: inputLinks.target.value
+    })
+  }
+
+  onSubmit() {}
+
   render() {
-    return <div>POIForm</div>
+    return (
+      <Form horizontal>
+        <PageHeader>
+          POI Input Form
+          <small>Complete each component below</small>
+        </PageHeader>
+
+        {/*
+          *********THIS IS MY POI NAME***********
+        */}
+        <FormGroup controlId="name">
+          <Col componentClass={ControlLabel} sm={2}>
+            POI Name
+          </Col>
+          <Col sm={10}>
+            <FormControl
+              type="text"
+              placeholder="Enter your POI name here"
+              value={this.state.name}
+              onChange={this.onChangeName}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY DATEPICKER***********
+        */}
+        <FormGroup>
+          <Col componentClass={ControlLabel} sm={2}>
+            POI Date
+          </Col>
+          <Col sm={6}>
+            <DatePicker
+              selected={this.state.startDate}
+              onChange={this.handleChange}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY DESCRIPTION AREA***********
+        */}
+        <FormGroup controlID="description">
+          <Col componentClass={ControlLabel} sm={2}>
+            POI description
+          </Col>
+          <Col sm={10}>
+            <FormControl
+              componentClass="textarea"
+              placeholder="Enter you POI description here"
+              value={this.state.description}
+              onChange={this.onChangeDescription}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY UPLOAD BUTTON***********
+        */}
+        <FormGroup controlID="chooseFile">
+          <Col componentClass={ControlLabel} sm={2}>
+            Upload Media
+          </Col>
+          <Col sm={10}>
+            <FormControl type="file" placeholder="Upload your files here" />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY LINKS TEXT AREA***********
+        */}
+        <FormGroup controlID="links">
+          <Col componentClass={ControlLabel} sm={2}>
+            POI Links
+          </Col>
+          <Col sm={10}>
+            <FormControl
+              componentClass="textarea"
+              placeholder="Enter related links here, separated by commas"
+              value={this.state.links}
+              onChange={this.onChangeLinks}
+            />
+          </Col>
+        </FormGroup>
+
+        {/*
+          *********THIS IS MY LIST OF STORY CHECKBOXES***********
+        */}
+        <FormGroup controlID="stories">
+          <Col componentClass={ControlLabel} sm={2}>
+            Stories
+          </Col>
+          <Col sm={10}>
+            <Checkbox>Story 1</Checkbox>
```

Comment:
Eventually, we will need to use the list of stories from the Api to render the list of options. For now I would recommend adding some stories to the dummy data so you can set it in place until the Api is ready. You should `map` over the list of stories.